tests/test_beamtime.py

```diff
@@ -0,0 +1,82 @@
+
+"""
+unit tests for beamtime info
+"""
+
+import datetime
+import os
+import socket
+import sys
+import unittest
+
+_test_path = os.path.dirname(__file__)
+_path = os.path.join(_test_path, '..')
+if _path not in sys.path:
+    sys.path.insert(0, _path)
+
+from tests.common import Capture_stdout, Capture_stderr
+
+from apstools.beamtime import bss_info
+
+
+def using_APS_workstation():
+    hostname = socket.gethostname()
+    return hostname.lower().endswith(".aps.anl.gov")
+
+
+class Test_Beamtime(unittest.TestCase):
+
+    def test_general(self):
+        self.assertEqual(bss_info.CONNECT_TIMEOUT, 5)
+        self.assertEqual(bss_info.POLL_INTERVAL, 0.01)
+        self.assertEqual(
+            bss_info.DM_APS_DB_WEB_SERVICE_URL,
+            "https://xraydtn01.xray.aps.anl.gov:11236")
+        self.assertIsNotNone(bss_info.api_bss)
+        self.assertIsNotNone(bss_info.api_esaf)
+
+    def test_iso2datetime(self):
+        self.assertEqual(
+            bss_info.iso2datetime("2020-06-30 12:31:45.067890"),
+            datetime.datetime(2020, 6, 30, 12, 31, 45, 67890)
+        )
+
+    def test_not_at_aps(self):
+        self.assertTrue(True)   # test something
+        if using_APS_workstation():
+            return
+
+        # do not try to test for fails using dm package, it has no timeout
+
+    def test_only_at_aps(self):
+        self.assertTrue(True)   # test something
+        if not using_APS_workstation():
+            return
+        runs = bss_info.api_bss.listRuns()
```

Comment:
Codacy found an issue: [local variable 'runs' is assigned to but never used (F841)](https://app.codacy.com/gh/BCDA-APS/apstools/pullRequest?prid=5764884)